CheckersCheckpoint/Game.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CheckersCheckpoint
+{
+    public class Game
+    {
+        public Game()
+        {
+            Board board = new Board();
+            board.GenerateCheckers();
+            board.DrawBoard();
+
+            Console.WriteLine("\nIf you want to move a checker one space diagonally forward, enter 'move'.");
+            Console.WriteLine("\nIf a jump is available for one of your checker, you must enter 'jump'.");
+
+            string choice = Console.ReadLine();
```

Comment:
Do I always have to move or jump, I cannot switch between the two during game play?